lab-kyle/model/client.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const uuid = require('node-uuid');
+
+const Client = function(socket) {
```

Comment:
Should probably just assign this function directly to `module.exports` so you don't have to worry about the Client property when you import it elsewhere.